Assets/Scripts/ScreenShotScript.cs

```diff
@@ -0,0 +1,73 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using UnityEditor;
+using System.Collections;
+
+public class ScreenShotScript : MonoBehaviour {
+
+	public AudioSource audio;
+	public RawImage screenShot;
+	
+	private Texture2D screenShotTexture;
+	private string lastScreenShotPath;
+	
+	// Use this for initialization
+	void Start () {
+		screenShotTexture = new Texture2D(100, 100);
+		screenShot.texture = screenShotTexture;
+	}
+	
+	// Update is called once per frame
+	void Update () {
+		
+	}
+	
+	void TakeScreenShot (string path) {
+		StartCoroutine (CaptureScreen (path));
+	}
+	
+	IEnumerator CaptureScreen (string path) {
+		// Wait till the last possible moment before screen rendering to hide the UI
+		yield return null;
+//		GameObject.Find("UICanvas").GetComponent<Canvas>().enabled = false;
```

Comment:
I want you not to comment out code but to delete it if it is no longer needed.
If you temporarily comment out code, please write the reason comment.